.github/workflows/build_windows_release_candidate.yml

```
@@ -0,0 +1,32 @@
+name: Build Windows Release Candidate
+
+on: [workflow_dispatch, workflow_call]
+
+jobs:
+  windows_release_build:
+    name: Build and Pack Windows Release Binaries
+    runs-on: windows-latest
+    timeout-minutes: 10
+    env:
+      SENTRY_DSN: ${{ secrets.SENTRY_DSN }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+
+      - name: Install Node
+        uses: actions/setup-node@v2.2.0
+        with:
+          node-version: 16
+
+      - name: Install NPM Dependencies
+        run: npm ci
+
+      - name: Build Windows Release Binaries
+        run: npm run action src/electron/build windows -- --buildMode=release
+
+      - name: Upload Windows Release Binaries
+        uses: actions/upload-artifact@v2
+        with:
+          name: client_windows_release_${{ github.sha }}
```

Let's specify in the name that this is not signed.

---

I meant in the file name:
```suggestion
          name: client_windows_release_unsigned_${{ github.sha }}
```

---

Also, I think this is the installer? Let's name it in a way that makes it clearer. Let's use `client_installer_windows_release_unsigned_` to match the debug build.